docs/best-practices/crate-node.txt

```diff
@@ -1,434 +1,482 @@
 
-* Start a new cluster and verify that it is healthy. This cluster may comprise
-  one or more brand-new master-eligible nodes, or may be an unsafely-bootstrapped
-  cluster formed as described above.
+* Start a new cluster and verify that it is healthy. The cluster may comprise
+  one or more new master-eligible nodes, or maybe an unsafely-bootstrapped
+  cluster, formed in the section
+  :ref:`crate-node-cli-re-purposing-nodes-unsafe-bootstrapping`
 
 * Stop all remaining data nodes.
 
-* On each data node, run the ``crate-node detach-cluster`` tool as shown
-  below. Verify that the tool reported Node was successfully detached from
-  the cluster.
+* On each data node, run the ``crate-node detach-cluster`` tool as demonstrated
+  below. Verify that the tool-reported node was successfully detached from the
+  cluster.
 
 * If necessary, configure each data node to discover the new cluster.
 
 * Start each data node and verify that it has joined the new cluster.
 
-* Wait for all recoveries to have completed, and investigate the data in the
-  cluster to discover if any was lost during this process.
+* Wait for all recoveries to complete, and investigate the data in the
+  cluster to find out if any data was lost during this process.
 
-The message Node was successfully detached from the cluster does not mean that
-there has been no data loss, it just means that tool was able to complete its job.
+The message `Node was successfully detached from the cluster` means that the
+detach-cluster tool successfully completed its job. It does not say that there
+has not been a data loss.
 
 Parameters
 ----------
 
-`repurpose`
-  Delete excess data when a node’s roles are changed.
+* ``repurpose``
+    Delete excess data when a node’s role is changed.
 
-`unsafe-bootstrap`
-  Specifies to unsafely bootstrap this node as a new one-node cluster.
+* ``unsafe-bootstrap``
+    Unsafely bootstrap this node as a new one-node cluster.
 
-`detach-cluster`
-  Specifies to unsafely detach this node from its cluster so it can join a
-  different cluster.
+* ``detach-cluster``
+    Specify to unsafely detach this node from its cluster so it can join
+    another cluster.
 
-`--ordinal <Integer>``
-  If there is more than one node sharing a data path then this specifies which
-  node to target. Defaults to 0, meaning to use the first node in the data path.
+* ``--ordinal <Integer>``
+    Specify which node to target if there is more than one node sharing a data
+    path. Defaults to 0, meaning to use the first node in the data path.
 
-`-C <KeyValuePair>`
-  Configures a setting.
+* ``-C <KeyValuePair>``
+    Configure a setting.
 
-`-h, --help`
-  Returns all of the command parameters.
+* ``-h, --help``
+    Return all of the command parameters.
 
-`-s, --silent`
-  Shows minimal output.
+* ``-s, --silent``
+    Show minimal output.
 
-`-v, --verbose`
-  Shows verbose output.
+* ``-v, --verbose``
+    Show verbose output.
 
 Examples
```

Comment:
this section appears to be entirely duplicate content

Reply:
Doesn't look like a duplicate to me really (Branch is up to date too).